bootstrap/interfaces/secret.go

```diff
@@ -68,4 +71,16 @@ type SecretProviderExt interface {
 
 	// IsJWTValid evaluates a given JWT and returns a true/false if the JWT is valid (i.e. belongs to us and current) or not
 	IsJWTValid(jwt string) (bool, error)
+
+	// HttpTransport returns the http.RoundTripper to be used by http-based clients
+	HttpTransport() http.RoundTripper
+
+	// SetHttpTransport sets the http.RoundTripper to be used by http-based clients
+	SetHttpTransport(rt http.RoundTripper)
+
+	// ZeroTrustEnabled returns whether zero trust principles are enabled
+	ZeroTrustEnabled() bool
```

Comment:
Match name used in the implementations
```suggestion
	IsZeroTrustEnabled() bool
```